core/include/gnuradio-4.0/BlockTraits.hpp

```diff
@@ -345,6 +345,12 @@ struct DummyPublishableSpan {
 };
 static_assert(PublishableSpan<DummyPublishableSpan<int>>);
 
+template<typename T>
+struct DummyPublishablePortSpan: public DummyPublishableSpan<T> {
```

Comment:
@drslebedev @wirew0rm I am wondering whether we should merge both concepts to create basically only one `PublishableSpan`. :thinking: